xlators/features/index/src/index.c

```diff
@@ -306,11 +308,7 @@ index_dir_create(xlator_t *this, const char *subdir)
 void
 index_get_index(index_priv_t *priv, uuid_t index)
 {
-    LOCK(&priv->lock);
-    {
-        gf_uuid_copy(index, priv->index);
-    }
-    UNLOCK(&priv->lock);
+    gf_uuid_copy(index, priv->index);
```

Comment:
nit: can we just call gf_uuid_copy() in index_link_to_base() directly, and eliminate index_get_index() altogether?

Reply:
done